client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -423,38 +427,52 @@ export const downloadBatchTallySheets = async (
     // Assume up until this point that we won't spill onto a second page. From here onward, no
     // longer make that assumption
 
-    // autoTable automatically adds page breaks
-    autoTable(doc, {
-      head: [['Candidates/Choices', 'Enter Stack Totals']],
-      body: contestChoices.map(choice => [
-        choice.name,
-        '', // Stack totals left blank for the audit board to fill out
-      ]),
-      startY: y,
-      margin: { bottom: pageMargin, left: pageMargin, right: pageMargin },
-      rowPageBreak: 'avoid',
-      theme: 'grid',
-      styles: {
-        cellPadding: tableCellPadding,
-        fillColor: 'white',
-        fontSize: defaultFontSize,
-        fontStyle: 'normal',
-        lineColor: 'black',
-        lineWidth: drawingLineWidth,
-        minCellWidth: tableCellMinWidth,
-        textColor: 'black',
-      },
-      headStyles: {
-        fontStyle: 'bold',
-      },
-    })
+    for (const [contestIndex, contest] of contests.entries()) {
+      // autoTable automatically adds page breaks
+      autoTable(doc, {
+        head: [
+          [{ content: contest.name, colSpan: 2 }],
+          ['Candidates/Choices', 'Enter Stack Totals'],
+        ],
+        body: contest.choices.map(choice => [
+          choice.name,
+          '', // Stack totals left blank for the audit board to fill out
+        ]),
+        startY: y,
+        margin: {
+          bottom: pageMargin,
+          left: pageMargin,
+          right: pageMargin,
+          top: pageMargin,
```

Comment:
Other than placing the `autoTable` call in a loop, the one other change that I've made here is the addition of this top margin. This makes sure that, if a table begins on a second page, it respects the second page's top margin. Before this PR, I'd been assuming that tables would always be wholly contained on a first page.